### PR TITLE
Update the PR guidelines in Contributing docs

### PR DESCRIPTION
**Pull request by tfny**
### What does this PR do?
This updates the guidelines for the contributing docs.  They have been approved by the c-suite, proxy team, and community team. 

### Motivation
To make it easier for the community to understand our PR process. 

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

 There seems to be something wrong with the spacing in lines 55-57.  I cannot figure it out.  Is it just me?

### CODE REVIEW

File: docs/content/contributing/submitting-pull-requests.md [hunk 1]
```diff
@@ -3,12 +3,214 @@ title: "Traefik Pull Requests Documentation"
...
+
+This will allow us to communicate directly and let you know if it is something we would accept. 
+It also allows us to make sure you have all the information you need during the design phase 
+so that it can be reviewed and merged quickly. 
+  
+If you have questions about the Triage process, [read more here](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).  
+  
+# The Pull Request Submit Process
+
+Merging a PR requires the following steps to be completed before the pull request is merged automatically.
```
Comment (moutoum) [inline docs/content/contributing/submitting-pull-requests.md:44]:
```suggestion
Merging a PR requires the following steps to be completed before it is merged automatically.
```

File: docs/content/contributing/submitting-pull-requests.md [hunk 2]
```diff
@@ -3,12 +3,214 @@ title: "Traefik Pull Requests Documentation"
...
+This label can be used when:
+* Updating a dependency
+* Merging branches back into the next version branch
+* Submitting minor documentation changes
+* Submitting changelog PRs
+
+# Why Was My Pull Request Closed?
+
+Traefik Proxy is made by the community for the community, 
+as such goal is to engage the community to make Traefik the best reverse proxy available.
```
Comment (mpl) [inline docs/content/contributing/submitting-pull-requests.md:123]: I don't get this sentence. maybe it's missing "the" before "goal"?

File: docs/content/contributing/submitting-pull-requests.md [hunk 3]
```diff
@@ -3,12 +3,214 @@ title: "Traefik Pull Requests Documentation"
...
+* Updating a dependency
+* Merging branches back into the next version branch
+* Submitting minor documentation changes
+* Submitting changelog PRs
+
+# Why Was My Pull Request Closed?
+
+Traefik Proxy is made by the community for the community, 
+as such goal is to engage the community to make Traefik the best reverse proxy available. 
+Unfortunately, part of this goal is maintaining a lean codebase and ensuring code velocity.
```
Comment (mpl) [inline docs/content/contributing/submitting-pull-requests.md:124]: why Unfortunately? these are good things.

File: docs/content/contributing/submitting-pull-requests.md (hunk 1 above)
Comment (tfny) [reply docs/content/contributing/submitting-pull-requests.md:49]: I think in this case we should be extra clear and repeat the term.

**Comment by mpl** [comment]
> I made the changes asked. How does it look?

yep, better. thanks.

**Review by tfny — COMMENTED** [review]
fixed the spelling

**Review by kevinpollet — APPROVED** [review]
LGTM

**Review by tomMoulard — APPROVED** [review]
LGTM :see_no_evil:

**Pull request merged**